jnigen/tool/pr_checks.dart

```diff
@@ -60,19 +81,18 @@ void main() async {
       .resolve("jni/lib/src/third_party/jni_bindings_generated.dart");
   final ffigenBindings = File.fromUri(ffigenBindingsPath);
   final oldBindingsText = ffigenBindings.readAsStringSync();
-  final ffigenCompare =
-      Runner("Generate & Compare FFIGEN bindings", tempJniPath)
-        ..chainCommand("dart", ["run", "ffigen", "--config", "ffigen.yaml"])
-        ..chainCallback("compare bindings", () async {
-          final newBindingsText = await ffigenBindings.readAsString();
-          if (newBindingsText != oldBindingsText) {
-            await ffigenBindings.writeAsString(oldBindingsText);
-            throw "new JNI.h bindings differ from old bindings";
-          }
-        });
+  final ffigenCompare = Runner("Generate & Compare FFIGEN bindings", jniPath)
+    ..chainCommand("dart", ["run", "ffigen", "--config", "ffigen.yaml"])
```

Comment:
I think a small issue is that if the generated bindings are different, old ones are not restored.

If this is problematic, I think adding a `finally` block which writes back the old bindings may solve the issue.

Reply:
I'll add it for completeness.